internal/authorization/subject_matcher.go

```diff
@@ -6,25 +6,31 @@ import (
 	"github.com/authelia/authelia/internal/utils"
 )
 
-func isSubjectMatching(subject Subject, subjectRule string) bool {
-	// If no subject is provided in the rule, we match any user.
-	if subjectRule == "" {
-		return true
-	}
+func isSubjectMatching(subject Subject, subjectRule []string) bool {
+	if len(subjectRule) > 0 {
```

Comment:
I think this condition is not necessary, you can remove it. We will gain one indentation level.

Reply:
Done. Thanks for your reviews!